common/scala/src/main/scala/org/apache/openwhisk/core/database/memory/MemoryArtifactStore.scala

```diff
@@ -285,8 +296,10 @@ class MemoryArtifactStore[DocumentAbstraction <: DocumentSerializer](dbName: Str
   }
 
   override def shutdown(): Unit = {
-    artifacts.clear()
-    attachmentStore.shutdown()
+    if (!ignoreShutdown) {
```

Comment:
worth commenting this - i dont understand why it would be necessary at face value.